lib/round.rb

```diff
@@ -0,0 +1,46 @@
+class Round 
+  attr_accessor :deck, 
+                :turns, 
+                :current_card, 
```

Comment:
`current_card` is a method you are defining on line 14, so we do not also want to have an `attr_reader` for it. It can lead to conflicts to have methods by the same name. In this case, I would remove it as an `attr_reader` altogether.